src/ccl_power.c

```diff
@@ -33,17 +70,15 @@ static void ccl_free_class_structs(ccl_cosmology *cosmo,
   if(init_arr[i_init--]) {
     if (spectra_free(sp) == _FAILURE_) {
       *status = CCL_ERROR_CLASS;
-      ccl_cosmology_set_status_message(
-        cosmo, "ccl_power.c: ccl_free_class_structs(): Error freeing CLASS spectra:%s\n", sp->error_message);
+      ccl_cosmology_set_status_message(cosmo, "ccl_power.c: ccl_free_class_structs(): Error freeing CLASS spectra:%s\n", sp->error_message);
```

Comment:
These long lines were changed to be easier to read for a reason. Let's put them back.

Reply:
I've broken up several of them.